universal-application-tool-0.0.1/app/views/applicant/ProgramIndexView.java

```diff
@@ -150,6 +150,7 @@ private ContainerTag programCard(
             .withText(applyText)
             .withId(baseId + "-apply")
             .withClasses(
+                "apply-button",
```

Comment:
Nit (non-blocking): You could define this constant in `view/style/ReferenceClasses.java`.

Reply:
Great suggestion! Updated